doc/source/tutorials/muppy_tutorial.rst

```diff
@@ -51,8 +51,8 @@ have a look at the objects, invoke an action, and look at the objects again.
 
    from pympler import tracker
 
-   self.memory_tracker = tracker.SummaryTracker()
-   self.memory_tracker.print_diff()
+   tracker.memory_tracker = tracker.SummaryTracker()
+   tracker.memory_tracker.print_diff()
```

Comment:
Your change injects a variable in the tracker module - for improving readability it'd be better to just get rid of `self.`.